src/component/Counter/Counter.test.js

```diff
@@ -0,0 +1,30 @@
+import { render, fireEvent } from "@testing-library/react";
+import Counter from "./Counter";
+
+describe("Counter Component", () => {
+  it("Should render with initial count", () => {
+    const { getByText } = render(<Counter initialValue={5} />);
+    const counterElement = getByText("Counter: 5");
+    expect(counterElement).toBeInTheDocument();
+  });
+
+  it("Should increment the count when the increment button is clicked", () => {
+    const { getByText } = render(<Counter initialValue={0} />);
+    const incrementButton = getByText("+");
+
+    fireEvent.click(incrementButton);
+
+    const counterElement = getByText("Counter: 1");
+    expect(counterElement).toBeInTheDocument();
```

Comment:
Your Code: expect(counterElement).toBeInTheDocument();
Suggestion: Simplified shortcut can be just check expect(counterElement)
Reason: If there is no match  jest will throw an exception 

Plz refer: https://testing-library.com/docs/react-testing-library/cheatsheet
![image](https://github.com/prateekd15/netflix-roulette/assets/22211840/edc549c4-4a04-4863-8be3-6b1a6be08548)

Reply:
Fixed as part of 903b467